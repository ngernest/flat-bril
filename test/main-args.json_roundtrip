{
  "functions": [
    {
      "args": [
        {
          "name": "x",
          "type": "int"
        },
        {
          "name": "y",
          "type": "int"
        }
      ],
      "instrs": [
        {
          "args": [
            "x",
            "y"
          ],
          "dest": "v",
          "funcs": [],
          "labels": [],
          "op": "add",
          "type": "int"
        },
        {
          "args": [
            "v"
          ],
          "funcs": [],
          "labels": [],
          "op": "print"
        }
      ],
      "name": "main"
    }
  ]
}
