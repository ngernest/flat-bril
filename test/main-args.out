@main
	op: add  		dest: v 		type: int		args: ["x", "y"]	
	op: print		args: ["v"]	
